.github/workflows/cd.yml

```diff
@@ -230,6 +242,41 @@ jobs:
           npm publish --access public
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Build wheels (linux)
+        if: matrix.build.PYPI_PUBLISH == true && matrix.build.NAME.startsWith('linux-')
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.build.TARGET }}
+          args: --manifest-path git-cliff/Cargo.toml --release --out dist
+          sccache: 'true'
+          manylinux: auto
```

Comment:
What does this `manylinux` option specify? I'm not so familiar with Python packaging stuff :/

Reply:
I haven't used it before (or if I have, I have forgotten), I mostly cribbed from other workflow examples. However, here's some info: [`manylinux`](https://github.com/pypa/manylinux) is a [platform compatibility tag](https://packaging.python.org/en/latest/specifications/platform-compatibility-tags/) for python packages.

From the project page:

> The various manylinux tags allow projects to distribute wheels that are automatically installed (and work!) on the vast majority of desktop and server Linux distributions.

Reply:
Got it, thanks for the info!

Reply:
Here's a bit more info: https://github.com/PyO3/maturin#manylinux-and-auditwheel